lib/rouge/lexer.rb

```diff
@@ -344,16 +344,16 @@ def as_list(val)
       end
     end
 
-    def as_lexer(val)
+    def as_lexer(val, opts={})
       return as_lexer(val.last) if val.is_a?(Array)
```

Comment:
Is there any chance that `opts` will be a non-empty hash? Does there need to be some mechanism to pass these through?

Reply:
This should be passed through, good catch. It's like this because Ruby's CGI parser will sometimes make stuff into singleton arrays.